client/layout/poll-invitation/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import snakeCase from 'lodash/snakeCase';
+import includes from 'lodash/includes';
+import memoize from 'lodash/memoize';
+let debug = require( 'debug' )( 'calypso:poll-invitation' );
+
+/**
+ * Internal dependencies
+ */
+let user = require( 'lib/user' )();
+import { ga as googleAnalytics } from 'analytics';
+import Gridicon from 'components/gridicon';
+import { tracks } from 'analytics';
+import config from 'config';
+import preferencesActions from 'lib/preferences/actions';
+
+const _preferencesKey = 'dismissedBrazilianSurvey';
+const _sectionWhiteList = [ 'reader', 'sites' ];
+
+function shouldDisplay() {
+	return 'pt-br' === user.get().localeSlug;
+}
+
+function recordEvent( eventAction ) {
+	googleAnalytics.recordEvent( 'Translator Invitation', eventAction );
+	let tracksEventName = 'calypso_poll_invitation_' + snakeCase( eventAction );
+	debug( 'recording event ' + tracksEventName );
+	tracks.recordEvent( tracksEventName );
+}
+
+let recordEventOnce = memoize( recordEvent );
```

Comment:
Yeah, I had the same thought process.  `once()` is a better match with what this one function is doing, but `memoize()` is a better match for what I actually want, even though I only use the resulting function for the one thing.

I could easily go the other way if it reads badly.